### PR TITLE
GLS edit of automated testing completed

### PR DESCRIPTION
**Pull request by nmuller66**
1. Please note: There are two comments that need review and the content fixed. 
The second one, under "The Game Begins Again" especially needs attention as it is not grammatically correct. I've made a suggestion with my comment in the file.

2. I would also suggest modifying the intro to the slide with the following graphic:
```
### DDD, BDD, and TDD
![TDD](images/bdd-tdd/ddd-bdd-tdd.jpg)
```
The intro has a punctuation error, a subject-verb agreement error ("the intersection ..provides),  and confuses the words "that" and "which".

Suggested edit:

"The intersection of TDD, DDD, and BDD provides a hybrid approach to development that assures predictable and productive outcomes."

### CODE REVIEW

File: docs/slides/content/automated-testing.md [hunk 1]
```diff
@@ -39,12 +38,11 @@ business concepts.
...
-* Basing complex designs on a model of the domain
-* Initiating a creative collaboration between technical and domain experts to iteratively 
-refine a conceptual model that addresses particular domain problems
-* Bridge the gap between domain experts and developers by using the same language to 
-create the same understanding
+* Places the project's primary focus on the core domain and domain logic
+* Bases complex designs on a model of the domain
+* Initiates a creative collaboration between technical and domain experts to iteratively refine a conceptual model to address domain problems
+* Bridges the gap between domain experts and developers by using the same language to create the same understanding
+<!--TechEdit: I'd say "to create a shared understanding.">
```
Comment (ckavili) [inline docs/slides/content/automated-testing.md:45]: feel free to update the sentence per your suggestion 😊

File: docs/slides/content/automated-testing.md [hunk 2]
```diff
@@ -249,10 +243,11 @@ Many times people will have built cool things that they didn't have tests for. W
 ### The Game Begins Again!
 
 ![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
- - As a table; Pick a project to build. Projects should be something complex that has lots of different kinds of things in it. Some examples if you're stuck could include _A Zoo_, _Shopping Center_, _a spaceship_ or _a city_.
+ - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
  - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
  - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->
-
+<!--The final bullet above doesn't make sense. What implementation? The features built previously by the pairs? How about 
+"The goal is to integrate the features that you built previously in pairs into the project selected by the table group." -->
```
Comment (ckavili) [inline docs/slides/content/automated-testing.md:250]: this sounds okay to me. Can we make it like this though?
```suggestion
"The goal is to integrate the elements that you built previously into the project selected by the table group." -->
```

**Comment by ckavili** [comment]
hey @nmuller66 thank you for your suggestions! I think we can definitely go with them so if you can just leave your updates and remove your comments, I can merge the PR 🙏

File: docs/slides/content/automated-testing.md [hunk 3]
```diff
@@ -39,12 +38,11 @@ business concepts.
...
-* Placing the project's primary focus on the core domain and domain logic
-* Basing complex designs on a model of the domain
-* Initiating a creative collaboration between technical and domain experts to iteratively 
-refine a conceptual model that addresses particular domain problems
-* Bridge the gap between domain experts and developers by using the same language to 
-create the same understanding
+* Places the project's primary focus on the core domain and domain logic
+* Bases complex designs on a model of the domain
+* Initiates a creative collaboration between technical and domain experts to iteratively refine a conceptual model to address domain problems
+* Bridges the gap between domain experts and developers by using the same language to create the same understanding
```
Comment (nmuller66) [inline docs/slides/content/automated-testing.md:44]:
```suggestion
* Bridges the gap between domain experts and developers by using the same language to create a shared understanding
```

File: docs/slides/content/automated-testing.md (hunk 1 above)
Comment (nmuller66) [inline docs/slides/content/automated-testing.md:45]:
```suggestion

```

File: docs/slides/content/automated-testing.md [hunk 4]
```diff
@@ -102,13 +97,12 @@ Based on the idea that multiple examples of specific cases convey information be
 
 ### Test Driven Development
 #### _What Is It?_
-Test Driven Development (TDD) is a software development process that relies on
-the repetition of a very short development cycle.
-Requirements are turned into test cases, where the software is developed to pass
+Test Driven Development (TDD) is a software development process that relies on the repetition of a very short development cycle.
```
Comment (nmuller66) [inline docs/slides/content/automated-testing.md:99]:
```suggestion
Test Driven Development (TDD) is a software development process that relies on short development cycles.
```

File: docs/slides/content/automated-testing.md [hunk 5]
```diff
@@ -102,13 +97,12 @@ Based on the idea that multiple examples of specific cases convey information be
...
+<!-- Tech Edit: I'm not sure what "the repetition of" means here. I'd suggest "...that relies on short development cycles" -->
```
Comment (nmuller66) [inline docs/slides/content/automated-testing.md:100]:
```suggestion

```

File: docs/slides/content/automated-testing.md [hunk 6]
```diff
@@ -249,10 +243,11 @@ Many times people will have built cool things that they didn't have tests for. W
 ### The Game Begins Again!
 
 ![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
- - As a table; Pick a project to build. Projects should be something complex that has lots of different kinds of things in it. Some examples if you're stuck could include _A Zoo_, _Shopping Center_, _a spaceship_ or _a city_.
+ - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
  - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
  - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->
-
+<!--The final bullet above doesn't make sense. What implementation? The features built previously by the pairs? How about
```
Comment (nmuller66) [inline docs/slides/content/automated-testing.md:248]:
```suggestion

```

File: docs/slides/content/automated-testing.md (hunk 2 above)
Comment (nmuller66) [inline docs/slides/content/automated-testing.md:249]:
```suggestion

```

File: docs/slides/content/automated-testing.md [hunk 7]
```diff
@@ -39,12 +38,11 @@ business concepts.
...
-* Basing complex designs on a model of the domain
-* Initiating a creative collaboration between technical and domain experts to iteratively 
-refine a conceptual model that addresses particular domain problems
-* Bridge the gap between domain experts and developers by using the same language to 
-create the same understanding
+* Places the project's primary focus on the core domain and domain logic
+* Bases complex designs on a model of the domain
+* Initiates a creative collaboration between technical and domain experts to iteratively refine a conceptual model to address domain problems
+* Bridges the gap between domain experts and developers by using the same language to create a shared understanding
+<!--TechEdit: I'd say "to create a shared understanding.">
```
Comment (nmuller66) [inline docs/slides/content/automated-testing.md]:
```suggestion

```

File: docs/slides/content/automated-testing.md (hunk 5 above)
Comment (nmuller66) [inline docs/slides/content/automated-testing.md]:
```suggestion

```

**Comment by nmuller66** [comment]
@ckavili Thank you! I've made the updates and removed my comments, but there are a couple of items that don't seem to be refreshing for me here in git. I've pulled the branch and checked and things look good. 
Can you take a look? I want to make sure I picked everything up and got the comments properly removed. :)

File: docs/slides/content/automated-testing.md [hunk 8]
```diff
@@ -249,10 +241,10 @@ Many times people will have built cool things that they didn't have tests for. W
 ### The Game Begins Again!
 
 ![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
- - As a table; Pick a project to build. Projects should be something complex that has lots of different kinds of things in it. Some examples if you're stuck could include _A Zoo_, _Shopping Center_, _a spaceship_ or _a city_.
+ - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
  - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
  - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->
-
+"The goal is to integrate the elements that you built previously into the project selected by the table group." -->
```
Comment (ckavili) [inline docs/slides/content/automated-testing.md:248]: a new line is required here :)

File: docs/slides/content/automated-testing.md [hunk 9]
```diff
@@ -249,10 +241,12 @@ Many times people will have built cool things that they didn't have tests for. W
 ### The Game Begins Again!
 
 ![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
- - As a table; Pick a project to build. Projects should be something complex that has lots of different kinds of things in it. Some examples if you're stuck could include _A Zoo_, _Shopping Center_, _a spaceship_ or _a city_.
+ - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
  - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
  - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->
 
+"The goal is to integrate the elements that you built previously into the project selected by the table group." -->
```
Comment (ckavili) [inline docs/slides/content/automated-testing.md:248]:
```suggestion
- The goal is to integrate the elements that you built previously into the project selected by the table group.<!-- {.element: class="fragment" data-fragment-index="3"} -->
```

File: docs/slides/content/automated-testing.md [hunk 10]
```diff
@@ -249,10 +241,12 @@ Many times people will have built cool things that they didn't have tests for. W
 ### The Game Begins Again!
 
 ![TDD](images/bdd-tdd/lego-tdd-2.png) <!-- {.element: class="inline-image" style="height:500px"} -->
- - As a table; Pick a project to build. Projects should be something complex that has lots of different kinds of things in it. Some examples if you're stuck could include _A Zoo_, _Shopping Center_, _a spaceship_ or _a city_.
+ - As a table group, pick a project to build. Projects must be something complex that has lots of different elements in it. Some examples, if you're stuck, might include a _zoo_, a _shopping center_, a _spaceship_, or a _city_.
  - All tests at the table must continue to pass at all times, regardless of which pair wrote them.  <!-- {.element: class="fragment" data-fragment-index="1"} -->
  - This time is that the implementation has to be integrated with the main project in the center of the table. <!-- {.element: class="fragment" data-fragment-index="2"} -->
```
Comment (ckavili) [inline docs/slides/content/automated-testing.md:247]:
```suggestion
```

**Comment by ckavili** [comment]
thank you!!! just two little things for the format :) then we are good to merge 🏃‍♀️ 💨

**Comment by nmuller66** [comment]
Great! Thanks! I've committed the suggestions. I am more aware of the line/slide break issue now so hopefully I won't make this mistake again!

**Pull request merged**